tests/runner/test_runner.py

```diff
@@ -20,13 +20,13 @@ class RunnerTestCase(unittest.TestCase):
     """A class for runner test cases."""
     def setUp(self):
         """Hook method for setting up the test fixture before exercising it."""
-        default_config_file = Path(__file__).parent / '../../superbench/config/default.yaml'
-        with default_config_file.open() as fp:
-            self.default_config = OmegaConf.create(yaml.load(fp, Loader=yaml.SafeLoader))
+        test_default_config_file = Path(__file__).parent / '../../tests/data/test_default.yaml'
```

Comment:
rename to test_config and test.yaml? no need for default

Reply:
updated.